docs/tutorials/add-environment-variable.md

```diff
@@ -0,0 +1,32 @@
+# Introducing new Environment Variable
+
+This document is part of the Appwrite contributors' guide. Before you continue reading this document make sure you have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/master/CODE_OF_CONDUCT.md) and the [Contributing Guide](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md).
+
+## Getting Started
+
+### Agenda
+Adding new features may require various configurations options to be set by the users. And for such options we use environment variables in Appwrite.
+
+This tutorial will cover, how to properly add new environment variable in Appwrite.
+
+### Naming environment varialbe
+The environment variables in Appwrite is prefixed with `_APP_`. Then if it belongs to specific cateogry, the category name is appended as `_APP_REDIS` for redis category. The available categories are General, Redis, MariaDB, InfluxDB, StatsD, SMTP, Storage and Functions. Finally the properly describing name is given to the variable as `_APP_REDIS_HOST` for redis host. You can find more information on available categories and existing environment variables at [environment variables doc](https://appwrite.io/docs/environment-variables).
```

Comment:
in Appwrite are *

Then if it belongs  <- remove the Then

 to a specific cateogry *

for the Redis category *